pkg/apis/scheduling/types.go

```diff
@@ -51,6 +51,9 @@ const (
 	// PodGroupInqueue means controllers can start to create pods,
 	// is a new state between PodGroupPending and PodGroupRunning
 	PodGroupInqueue PodGroupPhase = "Inqueue"
+
+	// PodGroupCompleted means all the pods of PodGroup are completed
+	PodGroupCompleted PodGroupPhase = "Completed"
```

Comment:
How about some Pods are completred but others are failed?

Reply:
My idea is not distinguish succeed & failure pods. In most common scenary, if successful + failed pods == total replicas, we set the PodGroup to Completed.